SmilesMiles/ViewModel/NewTripViewModel.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  NewTripViewModel.swift
+//  SmilesMiles
+//
+//  Created by Pavan Sai Nallagoni on 2/27/23.
+//
+
+import SwiftUI
+
+class NewTripViewModel: ObservableObject {
+    @Binding var trips: [String]
```

Comment:
Here also, please check if you can avoid the usage of binding

Reply:
Modified